src/api/nft-namespace.ts

```diff
@@ -473,6 +477,33 @@ export class NftNamespace {
     return getSpamContracts(this.config);
   }
 
+  /**
+   * Returns whether a contract is marked as spam or not by Alchemy. For more
+   * information on how we classify spam, go to our NFT API FAQ at
+   * https://docs.alchemy.com/alchemy/enhanced-apis/nft-api/nft-api-faq#nft-spam-classification.
+   *
+   * @param contractAddress - The contract address to check.
+   * @beta
```

Comment:
we don't need the `@beta` tags here since they are already released into the NFT API endpoints.